### PR TITLE
Ani development

### PR DESCRIPTION
**Pull request by AniMachaidze**

### CODE REVIEW

File: portfolio/src/main/webapp/career.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      </div>
+      <div class="footer">
+         <a href="https://www.linkedin.com/in/ani-m" class="fa fa-linkedin"></a>
+         <a href="https://github.com/AniMachaidze" class="fa fa-github"></a>
+         <a href="mailto:animachaidze@gmail.com" class="fa fa-google"></a>
+         <a href="https://www.facebook.com/ani.machaidze4" class="fa fa-facebook"></a>
+         <a href="https://www.instagram.com/ani.machaidze/?hl=en" class="fa fa-instagram"></a>
+      </div>
+   </body>
+</html>
```
Comment (astrelni) [inline portfolio/src/main/webapp/career.html:83]: Missing newline at end of file.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,69 @@
...
+      <div class="footer">
+         <a href="https://www.linkedin.com/in/ani-m" class="fa fa-linkedin"></a>
+         <a href="https://github.com/AniMachaidze" class="fa fa-github"></a>
+         <a href="mailto:animachaidze@gmail.com" class="fa fa-google"></a>
+         <a href="https://www.facebook.com/ani.machaidze4" class="fa fa-facebook"></a>
+         <a href="https://www.instagram.com/ani.machaidze/?hl=en" class="fa fa-instagram"></a>
+      </div>
+
+   </body>
+</html>
```
Comment (astrelni) [inline portfolio/src/main/webapp/index.html:69]: Missing newline here as well

File: portfolio/src/main/webapp/projects.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      <div class="footer">
+         <a href="https://www.linkedin.com/in/ani-m" class="fa fa-linkedin"></a>
+         <a href="https://github.com/AniMachaidze" class="fa fa-github"></a>
+         <a href="mailto:animachaidze@gmail.com" class="fa fa-google"></a>
+         <a href="https://www.facebook.com/ani.machaidze4" class="fa fa-facebook"></a>
+         <a href="https://www.instagram.com/ani.machaidze/?hl=en" class="fa fa-instagram"></a>
+      </div>
+
+   </body>
+</html>
```
Comment (astrelni) [inline portfolio/src/main/webapp/projects.html:50]: Missing newline

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,131 @@
...
+    padding: 25px;
+}
+
+/* Navigation menu style */
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #F3B5A7;
```
Comment (oydang) [inline portfolio/src/main/webapp/style.css:49]: Nit: all colors in CSS should be lowercase.

`#F3B5A7` -> `#f3b5a7`

https://google.github.io/styleguide/htmlcssguide.html#Capitalization

File: portfolio/src/main/webapp/career.html [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+            <li><a href="projects.html">Projects</a></li>
+            <li><a href="career.html">Career</a></li>
+            <li><a href="index.html">About</a></li>
+         </ul>
+      </div>
+
+      <div class="container">
+
+         <div class="title">
+            <h2> Work Experience</h2>
```
Comment (oydang) [inline portfolio/src/main/webapp/career.html:24]: Nit: extra whitespace in some places, including:

`<h2> Work Experience</h2>` -> `<h2>Work Experience</h2>`

`<h2> Education </h2>` -> `<h2>Education</h2>`

and in other places in this file.

**Pull request merged**